ansible-test/my-playbook.yml

```diff
@@ -0,0 +1,32 @@
+---
+- hosts: local-test
+  remote_user: root
+  tasks:
+    - name: add some config the .bashrc
+      become: yes
+      blockinfile:
+        path: /home/qujiabao/.bashrc
+        block: |
+          export DOCKER_HOST="0.0.0.0:2376"
+
+    - name: copy the daemon.json to /etc/docker
+      become: yes
+      copy: src=./daemon.json dest=/etc/docker
+
+    - name: copy the shell file to /usr/local/bin
+      become: yes
+      copy: src=./dockerd.sh dest=/usr/local/bin mode=a+x
+
+    - name: copy the service file to /etc/systemd/system
+      become: yes
+      copy: src=./dockerd.service dest=/etc/systemd/system mode=a+x
+
+    - name: force systemd to reread configs
+      become: yes
+      systemd: daemon_reload=yes
```

Comment:
This `reload` operation should only be performed if the configuration file is actually changed.

This is a good entry point to the topic of conditional execution in Ansible.